samples/boards/nrf52/mesh/onoff_level_app/README.rst

```diff
@@ -0,0 +1,101 @@
+.. _bluetooth-mesh-onoff-sample:
```

Comment:
These document labels must be unique across all the zephyr documents and this name is already used by samples/boards/nrf52/mesh/onoff-app/README.rst  (I suspect you copied that one to create this doc).  Change this document label to maybe

    .. _bluetooth-mesh-onoff-level-sample: